### DISCUSSION TITLE
guestbook

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#199] [General]
# guestbook

An online portfolio and blog by Theodorus Clarence. Showcase of my projects, and some of my thoughts about website development.

https://theodorusclarence.com/guestbook

**Comment by theodorusclarence** [comment]
Welcome to my guestbook 👋

You can sign in with GitHub to leave a comment 🥳

Your comment might be put in the testimonial section 😃

**Reply by alvinindra** [reply]
Hello~

**Reply by Jaiminkapopara** [reply]
heyy man, really enjoyed to go through your website. loved it

would you like to review mine 😉: https://jaimink.vercel.app/

**Reply by naseemkhandev** [reply]
Hello, really amaizing and very detailed
would you like to review mine ✌️: https://naseemkhan.dev

**Comment by muinmundzir** [comment]
Greetings Mas Theo, websitenya bagus ini inspirasi banget.
Bisa belajar banyak dari sini:D

**Comment by jason-a-fisher** [comment]
Hello Sir. Nice to see personal sites helping us React Nubs !

**Comment by rsipakov** [comment]
Hi! Nice project! But why didn't you use i18n like [next-i18next](https://github.com/isaachinman/next-i18next)?

**Reply by theodorusclarence** [reply]
I don't use it because I only need i18n on the blog post, and AFAIK next-18next library doesn't support markdown.
It's easier to implement it manually because I need to customize the slug to work with the view count database

**Reply by fahmiidris** [reply]
Please, make an article about this. I realy need it.

**Comment by rsipakov** [comment]


**Reply by theodorusclarence** [reply]
Sure, you can contact me via email as this guestbook is not meant for a discussion.

**Comment by sbmsr** [comment]
Keep it up Theo!

**Reply by theodorusclarence** [reply]
Thank you!

**Comment by jonwonglam** [comment]
Super impressive Theo! Your projects (including this website) are amazing for only starting web development in 2020. That library section with the snippets is a great idea

**Reply by theodorusclarence** [reply]
Thanks! 😀

**Reply by naseemkhandev** [reply]
Hey man, really amaizing and very detailed
would you like to review mine ✌️: https://naseemkhan.dev

**Comment by ghost** [comment]
Very impresive webiste and projects kak theo

**Comment by srisakkarin** [comment]
Hi

**Comment by ouedyan** [comment]
Your stuff is wonderful guy. From your templates, to your libraries and personal projects.

**Comment by blockchainchad** [comment]
GM friend! Thanks for doing what you do.

**Comment by AlijonovMukhammaddiyor** [comment]
Hey,  great work!

**Comment by DanylenkoDaryna** [comment]
Hello, Theodorus Clarence! Just checked in your useful post about choosing Next.js data fetching strategy, thank you for clearing this out for me!
I also checked out your site and home page. As a newbie in React, typescript, frontend, etc I am interested in the way how you made such amazing scrolling on your homepage: because I was trying to do the same, but was stuck on my way out. I was trying to add animation of pop-up content on my onScroll event but failed. Can you please give me some advice?
Thank you and have a good day!

**Comment by anthonykusuma** [comment]
I came across your website when I was looking for a comparison between SSG, SSR, and CSR. Thanks for the detailed comparison.

**Comment by kielllll** [comment]
Hello, Theodorus! Saw your post a while ago on Reddit and came across to your website. Looking forward to more content and best practices that you will post!

**Comment by michaelsynan** [comment]
Such a fun and impressive website, so glad I stumbled upon it :)

**Comment by nayyara-airlangga** [comment]
Hey man. Gotta say, I’m a big fan of your site and have been checking it on and off since 2021. It’s nice to see how you’ve progressed since then.

That said, Idk if someone’s told you yet or maybe you’ve considered it before but why not add a section for your resume? Not just included in a file that we can download, but like a timeline of your career path. I think it may or may not be a nice addition to your site.

Looking forward to more 🔥 stuff from you Theo

**Reply by theodorusclarence** [reply]
Thanks!

Of course, I've been considering adding that section but haven't had the time to do it. Maybe later 😀

**Comment by SamX23** [comment]
Hello . . .

Wow, man! why didn't I find this blog earlier?
This is a really informative react-related blog, I got more knowledge and insight from here, thanks a lot and keep up the good work !

**Reply by theodorusclarence** [reply]
Thanks! Glad you like it.

**Comment by b1gs4m** [comment]
great work!

**Reply by theodorusclarence** [reply]
Thank you!

**Comment by snake89322** [comment]
Nice to meet you! Your website is so awesome! I like it very much!

**Reply by theodorusclarence** [reply]
Thanks!

**Comment by MedX101** [comment]
Nice website TC, I was just wondering how do your get your cloudinary images to be displayed in this website..I have tried so many times but cannot find any way..did you an api ?

**Reply by theodorusclarence** [reply]
I'll reference this into an issue to answer :) https://github.com/theodorusclarence/theodorusclarence.com/issues/398

**Comment by MedX101** [comment]
Hey theo...is there any reason you chose not to included sitemap.xml and robots.txt from pushing them to production.

And how google are indexing your pages, are you submitting each and url of an post. I noticed put those (sitemap.xml and robots.txt ) under gitigonre

**Reply by theodorusclarence** [reply]
Hi! Since the discussion might get long, please [open an issue](https://github.com/theodorusclarence/theodorusclarence.com/issues/new) instead or ask in [AMA section](https://github.com/theodorusclarence/theodorusclarence.com/discussions/179)

**Comment by Wisesaturn** [comment]
That's a great blog! I'm getting a lot of insights.. and also I'm currently making a blog and I think it's a good reference.
Can I renew the blog layout by referring to it?

**Reply by theodorusclarence** [reply]
feel free to!

**Comment by criskell** [comment]
very nice!! thank you for your work

**Comment by bhatvikrant** [comment]
Hey there! This is one of my favourite technical blogs.

**Comment by Manish-Tamang** [comment]
Wow, it's so good! Love from Nepal 🇳🇵

**Comment by TEECOD3** [comment]
This is Beautiful

**Comment by hosseiniahb** [comment]
Hello dear Clarence.
It's really a beautiful and wonderful site that I get inspiration from and really enjoy.

**Comment by nelsonlaidev** [comment]
The new design is impressive, and I fully support your decision to make it closed-source, especially since my website has been copied by others multiple times. They have cloned my repository, made minimal edits to the text, and presented it as their own.

**Comment by fadhilkholaf** [comment]
Bang, kok lu keren banget si

**Comment by Jaiminkapopara** [comment]
heyy ma, really enjoyed to go through your website. loved it

would you like to review mine 😉: https://jaimink.vercel.app/

**Comment by naseemkhandev** [comment]
Hey, really amaizing and very detailed
would you like to review mine ✌️: https://naseemkhan.dev
